client/src/pages/GasCalculator.tsx

```diff
@@ -0,0 +1,422 @@
+          </div>
+
+          {/* Right column: Results - Wider */}
+          <div className="w-full lg:w-3/4 flex-grow">
+            <div className="border rounded-lg shadow-sm overflow-hidden bg-white h-full">
+              <div className="border-b p-3 bg-gray-50">
+                <h3 className="text-base font-medium">Results</h3>
+              </div>
+              
+              <div className="overflow-x-auto">
+                <table className="min-w-full">
+                  <thead>
+                    <tr className="bg-gray-50">
+                      <th className="p-2 border-b text-left text-xs font-medium text-gray-600 w-1/5">Metric</th>
+                      <th className="p-2 border-b text-left text-xs font-medium text-gray-600 w-1/5">
+                        <div className="flex items-center">
+                          <span className="w-2 h-2 bg-blue-400 rounded-full mr-1"></span>
+                          30d average
+                        </div>
+                      </th>
+                      <th className="p-2 border-b text-left text-xs font-medium text-gray-600 w-1/5">
+                        <div className="flex items-center">
+                          <span className="w-2 h-2 bg-green-400 rounded-full mr-1"></span>
+                          Low congestion
+                        </div>
+                      </th>
+                      <th className="p-2 border-b text-left text-xs font-medium text-gray-600 w-1/5">
+                        <div className="flex items-center">
+                          <span className="w-2 h-2 bg-red-400 rounded-full mr-1"></span>
+                          High congestion
+                        </div>
+                      </th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <tr className="hover:bg-gray-50">
+                      <td className="p-2 border-b text-xs text-gray-800">
+                        {proofCount === 1 && dayCount === 1 
+                          ? "Daily gas fee (FIL)" 
+                          : `Gas fee per day (FIL, for ${proofCount} proof${proofCount > 1 ? 's' : ''} per ${dayCount} day${dayCount > 1 ? 's' : ''})`}
+                      </td>
+                      <td className="p-2 border-b text-xs text-gray-800 font-mono">{metrics.gasAvg}</td>
+                      <td className="p-2 border-b text-xs text-gray-800 font-mono">{metrics.gasLow}</td>
+                      <td className="p-2 border-b text-xs text-gray-800 font-mono">{metrics.gasHigh}</td>
+                    </tr>
+                    <tr className="hover:bg-gray-50">
+                      <td className="p-2 border-b text-xs text-gray-800">
+                        {proofCount === 1 && dayCount === 1 
+                          ? "Daily PDP proof fee (FIL)" 
+                          : `PDP proof fee per day (FIL, for ${proofCount} proof${proofCount > 1 ? 's' : ''} per ${dayCount} day${dayCount > 1 ? 's' : ''})`}
```

Comment:
The default of 1 proof per 7 days should be 5 proofs per 1 day since that is the default in practice.

Reply:
Scratch that -- now that I know that "1 proof" really means 1 message with 5 proofs this default should be 1 proof per 1 day.